prod/onChainData128.8-validator1.yaml

```diff
@@ -290,3 +290,87 @@ EUROe-USD:
         params:
           id: euroe-stablecoin
           currency: USD
+
+
+WRBTC-DOC:
+  discrepancy: 1
+  precision: 8
+  heartbeat: 86400
+  trigger: 0.5
+  interval: 60
+  chains: [rootstock]
+  inputs:
+    - fetcher:
+        name: SovrynPriceFetcher
+        params:
+          base: '0x542fda317318ebf1d3deaf76e0b632741a7e677d'
+          quote: '0xe700691da7b9851f2f35f8b8182c69c53ccad9db'
+          amountInDecimals: 18
+
+WRBTC-rUSDT:
```

Comment:
I'm not sure if we should average the 2 prices of only take the one from the pool with the highes liquidity.
I believe we need to confirm if liquidities are similar to average the prices.